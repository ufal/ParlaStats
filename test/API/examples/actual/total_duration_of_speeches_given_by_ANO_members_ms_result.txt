[{'affiliation.person_id': 'PatrikNacher.1974', 'total_duration_ms': 2918040}, {'affiliation.person_id': 'VeraJourova.1964', 'total_duration_ms': 2329580}, {'affiliation.person_id': 'JosefKott.1970', 'total_duration_ms': 1763410}, {'affiliation.person_id': 'TatanaMala.1981', 'total_duration_ms': 1599290}, {'affiliation.person_id': 'KlaraDostalova.1971', 'total_duration_ms': 1540550}, {'affiliation.person_id': 'HelenaValkova.1951', 'total_duration_ms': 1401590}, {'affiliation.person_id': 'MonikaOborna.1989', 'total_duration_ms': 694250}, {'affiliation.person_id': 'JaroslavFaltynek.1962', 'total_duration_ms': 680770}, {'affiliation.person_id': 'MiloslavJanulik.1962', 'total_duration_ms': 650390}, {'affiliation.person_id': 'MilanHnilicka.1973', 'total_duration_ms': 645900}, {'affiliation.person_id': 'KristynaZelienkova.1974', 'total_duration_ms': 598810}, {'affiliation.person_id': 'DanTok.1959', 'total_duration_ms': 557780}, {'affiliation.person_id': 'BronislavSchwarz.1966', 'total_duration_ms': 506280}, {'affiliation.person_id': 'IvanPilny.1944', 'total_duration_ms': 473010}, {'affiliation.person_id': 'AlenaSchillerova.1964', 'total_duration_ms': 468780}, {'affiliation.person_id': 'JanaMrackovaVildumetzova.1973', 'total_duration_ms': 436170}, {'affiliation.person_id': 'RadekVondracek.1973', 'total_duration_ms': 321630}, {'affiliation.person_id': 'AndrejBabis.1954', 'total_duration_ms': 292960}, {'affiliation.person_id': 'JiriHlavaty.1948', 'total_duration_ms': 261000}, {'affiliation.person_id': 'JaroslavaPokornaJermanova.1970', 'total_duration_ms': 225160}, {'affiliation.person_id': 'JosefHajek.1956', 'total_duration_ms': 220690}, {'affiliation.person_id': 'RadimHolis.1975', 'total_duration_ms': 209080}, {'affiliation.person_id': 'BerenikaPestova.1971', 'total_duration_ms': 144780}, {'affiliation.person_id': 'PetrSadovsky.1969', 'total_duration_ms': 77520}, {'affiliation.person_id': 'RomanKubicek.1963', 'total_duration_ms': 73800}, {'affiliation.person_id': 'MilanBrazdil.1962', 'total_duration_ms': 52640}, {'affiliation.person_id': 'VeraAdamkova.1954', 'total_duration_ms': 36430}, {'affiliation.person_id': 'RadkaMaxova.1968', 'total_duration_ms': 7230}]

+--------------------------------+-------------------+
|     affiliation.person_id      | total_duration_ms |
+--------------------------------+-------------------+
|       PatrikNacher.1974        |      2918040      |
|        VeraJourova.1964        |      2329580      |
|         JosefKott.1970         |      1763410      |
|        TatanaMala.1981         |      1599290      |
|      KlaraDostalova.1971       |      1540550      |
|       HelenaValkova.1951       |      1401590      |
|       MonikaOborna.1989        |       694250      |
|     JaroslavFaltynek.1962      |       680770      |
|      MiloslavJanulik.1962      |       650390      |
|       MilanHnilicka.1973       |       645900      |
|    KristynaZelienkova.1974     |       598810      |
|          DanTok.1959           |       557780      |
|     BronislavSchwarz.1966      |       506280      |
|         IvanPilny.1944         |       473010      |
|     AlenaSchillerova.1964      |       468780      |
| JanaMrackovaVildumetzova.1973  |       436170      |
|      RadekVondracek.1973       |       321630      |
|        AndrejBabis.1954        |       292960      |
|        JiriHlavaty.1948        |       261000      |
| JaroslavaPokornaJermanova.1970 |       225160      |
|        JosefHajek.1956         |       220690      |
|        RadimHolis.1975         |       209080      |
|      BerenikaPestova.1971      |       144780      |
|       PetrSadovsky.1969        |       77520       |
|       RomanKubicek.1963        |       73800       |
|       MilanBrazdil.1962        |       52640       |
|       VeraAdamkova.1954        |       36430       |
|        RadkaMaxova.1968        |        7230       |
+--------------------------------+-------------------+
