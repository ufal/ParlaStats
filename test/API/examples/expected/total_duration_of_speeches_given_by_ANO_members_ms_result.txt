[{'affiliation.person_id': 'RadekVondracek.1973', 'total_duration_ms': 291840000.0}, {'affiliation.person_id': 'JaroslavaPokornaJermanova.1970', 'total_duration_ms': 137331790.0}, {'affiliation.person_id': 'AndrejBabis.1954', 'total_duration_ms': 95323344.0}, {'affiliation.person_id': 'JaroslavFaltynek.1962', 'total_duration_ms': 62572336.0}, {'affiliation.person_id': 'JanaMrackovaVildumetzova.1973', 'total_duration_ms': 39024576.0}, {'affiliation.person_id': 'AlenaSchillerova.1964', 'total_duration_ms': 27244690.0}, {'affiliation.person_id': 'KarelHavlicek.1969', 'total_duration_ms': 15647360.0}, {'affiliation.person_id': 'AdamVojtech.1986', 'total_duration_ms': 13979280.0}, {'affiliation.person_id': 'MartinKolovratnik.1975', 'total_duration_ms': 11743220.0}, {'affiliation.person_id': 'RichardBrabec.1966', 'total_duration_ms': 10657070.0}, {'affiliation.person_id': 'KlaraDostalova.1971', 'total_duration_ms': 8251750.0}, {'affiliation.person_id': 'HelenaValkova.1951', 'total_duration_ms': 4230210.0}, {'affiliation.person_id': 'JanVolny.1959', 'total_duration_ms': 3914380.0}, {'affiliation.person_id': 'JosefKott.1970', 'total_duration_ms': 3703800.0}, {'affiliation.person_id': 'DanTok.1959', 'total_duration_ms': 3655100.0}, {'affiliation.person_id': 'MartinStropnicky.1956', 'total_duration_ms': 3646530.0}, {'affiliation.person_id': 'VeraJourova.1964', 'total_duration_ms': 3494370.0}, {'affiliation.person_id': 'ZdenekSoukup.1952', 'total_duration_ms': 3380260.0}, {'affiliation.person_id': 'BronislavSchwarz.1966', 'total_duration_ms': 3342500.0}, {'affiliation.person_id': 'MarekNovak.1976', 'total_duration_ms': 3037740.0}, {'affiliation.person_id': 'JiriZlatuska.1957', 'total_duration_ms': 2756800.0}, {'affiliation.person_id': 'JiriBlaha.1961', 'total_duration_ms': 2298300.0}, {'affiliation.person_id': 'PatrikNacher.1974', 'total_duration_ms': 1891340.0}, {'affiliation.person_id': 'RobertPelikan.1979', 'total_duration_ms': 1476220.0}, {'affiliation.person_id': 'RadkaMaxova.1968', 'total_duration_ms': 1434500.0}, {'affiliation.person_id': 'MilanBrazdil.1962', 'total_duration_ms': 1387340.0}, {'affiliation.person_id': 'LubomirMetnar.1967', 'total_duration_ms': 1017200.0}, {'affiliation.person_id': 'MatejFichtner.1984', 'total_duration_ms': 917380.0}, {'affiliation.person_id': 'JosefHajek.1956', 'total_duration_ms': 893160.0}, {'affiliation.person_id': 'MiloslavJanulik.1962', 'total_duration_ms': 788900.0}, {'affiliation.person_id': 'RomanKubicek.1963', 'total_duration_ms': 732260.0}, {'affiliation.person_id': 'BerenikaPestova.1971', 'total_duration_ms': 478290.0}, {'affiliation.person_id': 'RobertKralicek.1977', 'total_duration_ms': 414070.0}, {'affiliation.person_id': 'PetrVrana.1975', 'total_duration_ms': 244040.0}, {'affiliation.person_id': 'DavidKasal.1969', 'total_duration_ms': 150220.0}, {'affiliation.person_id': 'OndrejBabka.1994', 'total_duration_ms': 108290.0}, {'affiliation.person_id': 'IvoVondrak.1959', 'total_duration_ms': 18300.0}, {'affiliation.person_id': 'JanaHanzlikova.1966', 'total_duration_ms': 0.0}, {'affiliation.person_id': 'JaroslavBzoch.1983', 'total_duration_ms': 0.0}]

Running time 27.9990234375 ms

+--------------------------------+-------------------+
|     affiliation.person_id      | total_duration_ms |
+--------------------------------+-------------------+
|      RadekVondracek.1973       |    291840000.0    |
| JaroslavaPokornaJermanova.1970 |    137331790.0    |
|        AndrejBabis.1954        |     95323344.0    |
|     JaroslavFaltynek.1962      |     62572336.0    |
| JanaMrackovaVildumetzova.1973  |     39024576.0    |
|     AlenaSchillerova.1964      |     27244690.0    |
|       KarelHavlicek.1969       |     15647360.0    |
|        AdamVojtech.1986        |     13979280.0    |
|     MartinKolovratnik.1975     |     11743220.0    |
|       RichardBrabec.1966       |     10657070.0    |
|      KlaraDostalova.1971       |     8251750.0     |
|       HelenaValkova.1951       |     4230210.0     |
|         JanVolny.1959          |     3914380.0     |
|         JosefKott.1970         |     3703800.0     |
|          DanTok.1959           |     3655100.0     |
|     MartinStropnicky.1956      |     3646530.0     |
|        VeraJourova.1964        |     3494370.0     |
|       ZdenekSoukup.1952        |     3380260.0     |
|     BronislavSchwarz.1966      |     3342500.0     |
|        MarekNovak.1976         |     3037740.0     |
|       JiriZlatuska.1957        |     2756800.0     |
|         JiriBlaha.1961         |     2298300.0     |
|       PatrikNacher.1974        |     1891340.0     |
|       RobertPelikan.1979       |     1476220.0     |
|        RadkaMaxova.1968        |     1434500.0     |
|       MilanBrazdil.1962        |     1387340.0     |
|       LubomirMetnar.1967       |     1017200.0     |
|       MatejFichtner.1984       |      917380.0     |
|        JosefHajek.1956         |      893160.0     |
|      MiloslavJanulik.1962      |      788900.0     |
|       RomanKubicek.1963        |      732260.0     |
|      BerenikaPestova.1971      |      478290.0     |
|      RobertKralicek.1977       |      414070.0     |
|         PetrVrana.1975         |      244040.0     |
|        DavidKasal.1969         |      150220.0     |
|        OndrejBabka.1994        |      108290.0     |
|        IvoVondrak.1959         |      18300.0      |
|      JanaHanzlikova.1966       |        0.0        |
|       JaroslavBzoch.1983       |        0.0        |
+--------------------------------+-------------------+
